examples/Linux/README.md

```diff
@@ -0,0 +1,106 @@
+# wolfSentry lwIP Echo Test
+
+This is an extremely basic demo application that uses wolfSentry with the lwIP stack via PCAP on Docker to echo anything sent to it. wolfSentry will filter various types of traffic which can be tested using three Docker nodes the test generates.
+
+## Prerequisites
+
+This test is designed to be run on Linux or macOS, but should work on Windows as well. You need the following installed on your computer:
+
+* Docker - <https://docs.docker.com/get-docker/>
+* docker-compose - <https://docs.docker.com/compose/install/>
+
+## Running echo server
+
+The following command will build the test echo server and start this up along with three testing nodes:
+
+```sh
+sudo docker-compose -f docker-compose.yml up
+```
+
+You can follow the logs for the echo server using:
+
+```sh
+sudo docker-compose -f docker-compose.yml logs -f
+```
+
+It is recommended that you keep the logs following running whilst running the tests.
+
+## Testing
+
+### Accessing nodes
+
+There are three user test nodes to play with. They are named `wolfsentry-test_tester?_1` where `?` is 1, 2 or 3. To log into tester2 as an example:
+
+```sh
+sudo docker exec -it wolfsentry-test_tester2_1 /bin/sh
+```
+
+### Node details
+
+#### Echoserver
+
+* IP address: 172.20.20.3 (node) 127.20.20.5 (echo process)
+* MAC address: de:c0:de:01:02:03
+
+The echo test process runs from this node, it uses PCAP and lwIP to create a static IP of 127.20.20.5 for the actually test.
+
+#### Tester 1
+
+* IP address: 172.20.20.10
+* MAC address: de:c0:de:03:02:01
+
+The sentry test is configured to allow this node to ping the echoserver node, but not TCP connect.
+
+#### Tester 2
+
+* IP address: 172.20.20.20
+* MAC address: de:c0:de:03:02:02
+
+The sentry test is configured to block this node pinging the echoserver node, but allow TCP connect.
```

Comment:
TCP connect is correctly allowed by MAC address, per the server logs and the `nc` result.

The ICMP ping is allowed, and the server isn't showing any log activity for the ICMP traffic.

Reply:
Again, I cannot reproduce this. But I'm guessing ICMP is handled by something else which I'll need to investigate.